source/backend/api/Areas/Acquisition/Controllers/ReportController.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Pims.Api.Areas.Acquisition.Models.Search;
+using Pims.Api.Helpers.Constants;
+using Pims.Api.Helpers.Exceptions;
+using Pims.Api.Helpers.Extensions;
+using Pims.Api.Helpers.Reporting;
+using Pims.Api.Models;
+using Pims.Api.Policies;
+using Pims.Api.Services;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Areas.Acquisition.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("acquisitionfiles")]
+    [Route("v{version:apiVersion}/[area]/export")]
+    [Route("[area]/export")]
+    public class ReportController : ControllerBase
+    {
+        private readonly IAcquisitionFileService _acquisitionService;
+        private readonly ILogger _logger;
+
+        public ReportController(IAcquisitionFileService acquisitionService, ILogger<ReportController> logger)
+        {
```

Comment:
@eddherrera - we have a separate folder for "reports" - we put them under `api/Areas/Reports/Controllers/AcquisitionController.cs`. Please consider moving it there (or merging with the one controller from Devin's PR once that gets merged

Reply:
Ok, I'll merge after his.

Reply:
relocated